sdk/formrecognizer/azure-ai-formrecognizer/tests/test_dac_analyze_layout.py

```diff
@@ -163,6 +163,34 @@ def test_layout_multipage_table_span_pdf(self, client):
         assert layout.tables[2].row_count == 24
         assert layout.tables[2].column_count == 5
 
+    @pytest.mark.live_test_only
+    @skip_flaky_test
+    @FormRecognizerPreparer()
+    @DocumentAnalysisClientPreparer()
+    @recorded_by_proxy
+    def test_layout_url_annotations(self, client):
+        poller = client.begin_analyze_document_from_url("prebuilt-layout", self.annotations_url_jpg)
+        layout = poller.result()
+        assert len(layout.pages) > 0
+        assert len(layout.pages[0].annotations) > 0
+        assert layout.pages[0].annotations[0].kind == "check"
+        assert layout.pages[0].annotations[0].polygon
+        assert layout.pages[0].annotations[0].confidence > 0.5
```

Comment:
should we just assert that `confidence` is not None? Or is `confidence` usually pretty stable with FR?

Reply:
Confidence does vary but this one is currently at around 0.7 I dont think it'll get close to 0.5 unless the service breaks a lot...so I think checking for a confidence above 0.5 should be fine